gen/field.go

```diff
@@ -22,10 +22,19 @@ package gen
 
 import (
 	"fmt"
+	"reflect"
+	"strings"
 
 	"go.uber.org/thriftrw/compile"
 )
 
+const (
+	goTagName      = "go.tag"
+	goTagDelimiter = " "
+	goTagSep       = ":"
+	jsonTagName    = "json"
```

Comment:
I would drop the `goTagDelimiter` and `goTagSep` constants. They hurt readability.